receiver/oktareceiver/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package oktareceiver provides a receiver that receives telemetry from an Okta domain.
+package oktareceiver // import "github.com/observiq/bindplane-agent/receiver/oktareceiver"
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"go.opentelemetry.io/collector/config/configopaque"
+)
+
+var (
+	defaultPollInterval = time.Minute
+
+	// OktaTimeFormat ISO 8601 Format
+	OktaTimeFormat = "2006-01-02T15:04:05Z"
+
+	// oktaMaxLimit maximum number of log objects returned in one call to Okta API
+	oktaMaxLimit = 1000
```

Comment:
I'd consider moving this over to logs.go since that's the only place it's referenced